dashboard/db/migrate/20171005192949_add_user_id_to_pd_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToPdApplications < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :pd_applications, :user, after: :id, null: false, index: true
```

Comment:
what does `after` mean? I can't find an explanation of it in the documentation for add_reference

Reply:
It's part of the [column](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb#L151) schema definitions, not directly part of `add_reference`. 

It adds the new column `after` the specified column, in this case after `id`, or as the second (as opposed to last) column in that table's schema.

Strangely I can't find it in the official Rails docs, but it has been in the codebase for a long time. Here's an old stack overflow post: https://stackoverflow.com/a/3251000/7794941